calico_rkt/calico_policy.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from constants import *
+from calico_rkt import configure_logging
+from pycalico.datastore import DatastoreClient
+from pycalico.datastore_datatypes import Rule, Rules
+
+KUBERNETES_DEFAULT_PROFILE = "default_profile"
+
+_log = logging.getLogger(__name__)
+configure_logging(_log)
+
+class BasePolicyPlugin(object):
+    def __init__(self, network_config, env):
```

Comment:
In other words, these aren't CNI plugins (which is why I'd call them driver's instead of plugins), and so they shouldn't need to be aware of CNI specific concepts, like CNI_ARGS, etc.